src/python/grpcio/grpc/_cython/_cygrpc/aio/iomgr/io_loop.pyx.pxi

```diff
@@ -0,0 +1,83 @@
+# Copyright 2019 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import sys
+import threading
+
+_SECONDS_PER_MILLISECOND = 0.001
+cdef _IOLoop _io_loop = None
+
+cdef class _IOLoop:
+
+    def __init__(self):
+        global _io_loop
+
+        # Only one instantiation is exepected        
+        assert _io_loop is None
```

Comment:
I'd add something similar to what the comment says to the assertion message, to make it more understanable for developers
```suggestion
        assert _io_loop is None, "More than one loop found"
```